app/controllers/user/swaps_controller.rb

```diff
@@ -38,6 +41,13 @@ def destroy
 
   private
 
```

Comment:
We should not get here normally, because the UI has some filters and won't show you swaps unless your mobile phone number has been verified. Just in case